thirdparty/vulkan/loader/loader.c

```diff
@@ -63,6 +69,11 @@
 #include <cfgmgr32.h>
 #include <initguid.h>
 #include <devpkey.h>
+#include <winternl.h>
+#include <d3dkmthk.h>
```

Comment:
This makes things tricky for cross-compilation from Linux as this header is not part of mingw but is provided by WINE. In my distro it's at `/usr/include/wine/windows/ddk/d3dkmthk.h` which is not in the default include paths, so I'll have to hardcode this extra include path.

Reply:
And that header requires `/usr/include/wine/windows/d3dukmdt.h` too, so another path to hardcode \o/

Reply:
Fun, on Debian-based distros the `wine` part of the path is duplicated: https://packages.ubuntu.com/disco/amd64/libwine-dev/filelist

And it actually doesn't build even then, I get a ton of build errors typical to windows headers being included in the wrong order...